share/jupyter/kernels/xr/kernel.json.in

```diff
@@ -1,5 +1,5 @@
 {
-  "display_name": "R 4.3.1 (xr)",
+  "display_name": "R 4.4.1 (xr)",
```

Comment:
Is it possible to change this to reflect the version installed in the environment? Something like
```json
"display_name": "R @R_VERSION_MAJOR@.@R_VERSION_MINOR@ (xr)",
```

Reply:
Sure !